gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_20_0/schema.yml

```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.